pkg/lib/bundle/validate_test.go

```diff
@@ -61,6 +61,14 @@ func TestValidateBundleDependencies(t *testing.T) {
 				fmt.Errorf("couldn't parse dependency of type olm.crd"),
 			},
 		},
+		{
+			description: "registryv1 bundle/invalid label type",
+			mediaType:   RegistryV1Type,
+			directory:   "./testdata/validate/invalid_dependencies_bundle/invalid_label_dependency/",
+			errs: []error{
+				fmt.Errorf("couldn't parse dependency of type olm.label"),
+			},
+		},
```

Comment:
Should we also include a test for a valid and invalid label property?

Reply:
This validation is for dependency given dependency information is provided via a separate file. All of the properties are extracted from CSV hence I added the labels into the example CSV in `/manifests`.